SignalServiceKit/src/Util/OWSBackgroundTask.m

```diff
@@ -1,5 +1,5 @@
 //
-//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
 #import "OWSBackgroundTask.h"
```

Comment:
We could refactor `OWSBackgroundTask` to only create a single background task from the POV of iOS.